### ISSUE TITLE
3.9.0 try to check js file with vue rules and broke down

### ISSUE THREAD

**Issue by deangoku** [vuejs/eslint-plugin-vue#125]
**Tell us about your environment**

* **ESLint Version: 4.3.0
* **eslint-plugin-vue Version: 3.9.0
* **Node Version: 6.11.2

**Please show your full configuration:**

```
// http://eslint.org/docs/user-guide/configuring
module.exports = {
  root: true,
  // https://github.com/feross/standard/blob/master/RULES.md#javascript-standard-style
  extends: ['standard', 'plugin:vue/recommended'],
  parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },
  env: {
    browser: true
  },
  // add your custom rules here
  rules: {
    // allow paren-less arrow functions
    'arrow-parens': 0,
    // allow async-await
    'generator-star-spacing': 0,
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
    // allow shorthand for method definitions
    'space-before-function-paren': 0
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```bash
npm up
```

**What did you expect to happen?**
eslint-plugin-vue upgrade from 3.8.0 to 3.9.0 and work well
**What actually happened? Please include the actual, raw output from ESLint.**
totally broke down. 3.9.0 try to check js file with vue rules……
![image](https://user-images.githubusercontent.com/6375486/28951045-16c816bc-78fb-11e7-9975-9069dc3fba86.png)

**Comment by mysticatea** [comment]
Thank you for the report.

However, I could not reproduce it.
Could you tell me the repro steps?

**Comment by afontcu** [comment]
Hi,

I don't know if this is related, but since I upgraded to 3.9.0 the `Use the latest vue-eslint-parser` message appeared again even though I already placed babel-eslint inside the `parserOptions` object (and it was working properly with prior versions of eslint-plugin-vue).

Thanks!

**Comment by mysticatea** [comment]
@afontcu That was it. I will fix. Thank you.

**Comment by michalsnik** [comment]
Fixed in `v3.10.0`. Let us know if you still have any issues.
